tests/general/SubtypingTest.java

```diff
@@ -15,19 +15,19 @@ void test(
             @AllowedProviders("p1") String i) {
         @UnknownAlgorithmOrProvider String a = x;
         @UnknownAlgorithmOrProvider String b = y;
-        // :: error: assignment.type.incompatible :: error: allowed.algorithm.or.provider.not.set
```

Comment:
Yes it was changed to a shorter error key: https://github.com/typetools/checker-framework/commit/5c04d6836d8e65daf79667de6539e0d5c055fbbb